avm/res/azure-stack-hci/cluster/tests/e2e/3nodeswitchless.defaults/main.test.bicep

```diff
@@ -35,7 +35,7 @@ param arbDeploymentSPObjectId string = ''
 #disable-next-line secure-parameter-default
 param arbDeploymentServicePrincipalSecret string = ''
 
-@description('Optional. The service principal ID of the Azure Stack HCI Resource Provider. If this is not provided, the module attemps to determine this value by querying the Microsoft Graph.')
+@description('Required. The service principal ID of the Azure Stack HCI Resource Provider in this tenant. To find this value, look up the service principal with app ID 1412d89f-b8a8-4111-b4fd-e82905cbd85d after registering the Azure Stack HCI RP in the tenant. For example: `Get-AzADServicePrincipal -ApplicationId 1412d89f-b8a8-4111-b4fd-e82905cbd85d`.')
```

Comment:
Hey @mbrat2005, what's the name of that application? I can't find an application with that ID (neither in the portal, nor via code)